src/utils/read-file.ts

```diff
@@ -0,0 +1,103 @@
+import fs from 'fs'
+import glob from 'glob'
+import yaml from 'js-yaml'
+import R from 'ramda'
+
+enum FileExtension {
+  JSON = 'json',
+  YAML = 'yaml',
+  YML = 'yml',
+}
+type File = {
+  contents: {
+    properties: {
+      runtimeEnvironment: string
+    }
+    required: string[]
+    sops: Record<string, any>
+  }
+  filePath: string
+}
+
+class NoFileError extends Error {
```

Comment:
interesting, is this the common way to create your own errors in typescript?

Reply:
Think so https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#example. Felt weird about that too

Reply:
ok